includes/emails/class-wc-email.php

```diff
@@ -560,7 +561,7 @@ public function style_inline( $content ) {
 			if ( $this->supports_emogrifier() ) {
 				$emogrifier_class = '\\Pelago\\Emogrifier';
 				if ( ! class_exists( $emogrifier_class ) ) {
-					include_once dirname( dirname( __FILE__ ) ) . '/libraries/class-emogrifier.php';
+					return '<style type="text/css">' . $css . '</style>' . $content;
```

Comment:
Just a nitpick but what do you think if, instead of duplicating this line https://github.com/woocommerce/woocommerce/pull/25525/files#diff-176d516db33c360b187bf2539bb465c5R574 here, we change the first if condition to check if the class exist. This should simplify a bit the code. Here is what I have in mind:

```php
			$emogrifier_class = '\\Pelago\\Emogrifier';

			if ( $this->supports_emogrifier() && class_exists( $emogrifier_class ) ) {
				try {
					$emogrifier = new $emogrifier_class( $content, $css );
					$content    = $emogrifier->emogrify();
				} catch ( Exception $e ) {
					$logger = wc_get_logger();
					$logger->error( $e->getMessage(), array( 'source' => 'emogrifier' ) );
				}
			} else {
				$content = '<style type="text/css">' . $css . '</style>' . $content;
			}
```